bot/events/badbotHandler.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
+const source = require('rfr');
+
+const { sendMessage } = source('bot/utils/util');
+const logger = source('bot/utils/logger');
+
+const { Responses } = source('models/responses');
+const { Statistics, EventTypes } = source('models/statistics');
+
+const badBotRegex = /[Bb][Aa][Dd]\s+[Bb][Oo][Tt]/;
+
+function botMessage(message) {
+    if (message.guild === null || message.author.bot) return;
+
+    if (!badBotRegex.test(message.content)) return;
+
+    return Responses.findForMode(message.guild.id, 'badbot')
+        .then((res) => {
+            const response = res.length !== 0 ? _.sample(res).message : ':\'[';
```

Comment:
Put `':\'['` into a field variable at the top of the file, called default message